native/cocos/ui/edit-box/EditBox-ios.mm

```diff
@@ -234,12 +252,32 @@ - (void)textViewDidChange:(UITextView *)textView {
     callJSFunc("input", [textView.text UTF8String]);
 }
 @end
+
+
+@interface CustomUITextField: UITextField
+@property (nonatomic, assign) BOOL canBecomeFirstResponderDynamically;
+@end
+
+@implementation CustomUITextField
+- (instancetype)initWithFrame:(CGRect)frame {
+    self = [super initWithFrame:frame];
+    if (self) {
+        _canBecomeFirstResponderDynamically = YES;
+    }
+    return self;
+}
+
+- (BOOL)canBecomeFirstResponder {
+    return self.canBecomeFirstResponderDynamically;
+}
```

Comment:
Why need this function as there is a property named canBecomeFirstResponderDynamically?

Reply:
I make a mistake, canBecomeFirstResponderDynamically's default value is NO